database-commons/src/main/java/io/cdap/plugin/db/source/AbstractDBSource.java

```diff
@@ -383,7 +383,7 @@ protected Class<? extends DBWritable> getDBRecordType() {
    * @return external documentation link
    */
   protected String getExternalDocumentationLink() {
-    return null;
+    return "https://en.wikipedia.org/wiki/SQLSTATE";
```

Comment:
is this method still needed? can we remove it?

Reply:
Not needed now. Removed. 7c8815dc3849a51d85d2b8624f75aa866a66a56c